packages/blade/src/utils/mapProps/a11yMap.ts

```diff
@@ -0,0 +1,93 @@
+export const accessibilityValue = {
+  accessibilityValueMax: 'aria-valuemax',
+  accessibilityValueMin: 'aria-valuemin',
+  accessibilityValueNow: 'aria-valuenow',
+  accessibilityValueText: 'aria-valuetext',
+};
```

Comment:
why not have map for `.web` and `.native`? with that you can just have one mapProps file and then the specific platforms will render that a11y key

Reply:
Actually we are not doing bi-directional mapping. 

Say you have `accessibilityLabel=hello` 
In native this prop will just be directly be returned,
but in web this will be converted to `aria-label`

We are not doing aria-label -> accessibilityLabel conversion.

Reply:
Correct. That's what I'm suggesting, instead of doing the map and changing things there why not create 2 maps in 2 files(`.native`, `.web`)

so you can have 2 files 

```
// accessibilityProps.native.ts

export const accessibilityValue = {
  accessibiltyRole: 'accessibiltyRole',
  accessibilityLabel: 'accessibilityLabel',
};


// accessibilityProps.web.ts

export const accessibilityValue = {
  accessibiltyRole: 'role',
  accessibilityLabel: 'aria-label',
};
```
with this you'll get rid of all the map functions you have to do this programatically.

Reply:
I see, okay I'll try this approach out and see which works best. 

Although 

```ts
export const accessibilityValue = {
  accessibiltyRole: 'accessibiltyRole',
  accessibilityLabel: 'accessibilityLabel',
};
```

is a bit redundant + now we need to maintain two accessibility maps.

Reply:
Can we remove the word `accessibility` from the keys of `mapA11yProps` & call it `makeAccessible`?

Something like this:
```
makeAccessible({ role, label, checked, valueText, valueMin, valueMax })
```

Reply:
atleast it's understandable IMO compare to reading all those maps and iterations

Reply:
> makeAccessible({ role, label, checked, valueText, valueMin, valueMax })

It's fine but this will clash with lot of other props if we ever create a wrapper component like I proposed in <Box \/> 
Plus I want to make it similar to how react-native calls them.